paasta_tools/deployd/master.py

```diff
@@ -245,6 +247,7 @@ def prioritise_bouncing_services(self):
             self.inbox_q.put(ServiceInstance(
                 service=service,
                 instance=instance,
+                cluster=self.config.get_cluster(),
                 watcher=self.__class__.__name__,
                 bounce_by=int(time.time()),
                 bounce_timers=None,
```

Comment:
Wouldn't here be a good place to do priority=self.config.get_bounce_priority? 

Is the inbox_q not a priority queue?